plugins/gatsby-remark-dvc-linker/simpleLinker.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env node */
+
+const { createLinkNode } = require('./helpers')
+
+const entries = [
```

Comment:
better to move it into some configuration file

Reply:
Sure, where you do think it should go? I'd assume at the root like `redirects-list.json`. Maybe `linked-terms.js`?

Reply:
we can make part of `content/` I guess ... along the community.json?

Reply:
Sure!